frontend/www/js/omegaup/components/problem/Settings.vue

```diff
@@ -4,10 +4,10 @@
       <div class="form-group col-md-6">
         <label>{{ T.problemEditFormLanguages }}</label>
         <select
-          v-model="languages"
-          name="languages"
+          v-model="currentLanguages"
+          name="current_languages"
```

Comment:
```suggestion
          name="languages"
```

esto creo que también se usa en el servidor (tenemos una forma que hace un POST de verdad), así que es mejor preservar el nombre original.

Reply:
No entendí esto, sugieres que cambie ambas lineas o sólo la 8 ¿?

Reply:
sólo la 8 (la línea anterior está ahí para dar contexto)

Reply:
Listo :)